### ISSUE TITLE
spark:dataframe does not work on Java 11

### ISSUE THREAD

**Issue by ppalaga** [apache/camel-quarkus#1955]
Blocked by https://issues.apache.org/jira/browse/CAMEL-15650

**Comment by ppalaga** [comment]
Won't fix, because we plan to remove the existing JVM-only Spark extension https://github.com/apache/camel-quarkus/issues/3466
